modules/core/src/image/vpCannyEdgeDetection.cpp

```diff
@@ -235,7 +235,7 @@ vpCannyEdgeDetection::performFilteringAndGradientComputation(const vpImage<unsig
 /**
  * \brief Get the interpolation weights and offsets.
  *
- * \param[in] absoluteTheta : The absolute value of the angle of the edge, expressed in degrees.
+ * \param[in] positiveTheta : The positive value of the angle of the edge, expressed in degrees.
```

Comment:
To be complete, you could precise that this angle is between -180 and +180 deg

Reply:
Is there a goo reason to consider degrees instead of rad?
antan2() is returning rad
You can use `M_PI_2` and `M_PI_4` macros

Reply:
I suggest to rename `positiveTheta` into `gradientOrientation` to be more explicit

Reply:
These suggestions have been addressed in the new commit e459437c747354287392d4a6761cb5393aaea853